src/js/components/Select/__tests__/Select-test.js

```diff
@@ -1714,3 +1713,38 @@ describe('Select', () => {
   window.scrollTo.mockRestore();
   window.HTMLElement.prototype.scrollIntoView.mockRestore();
 });
+
+describe('applyKey function', () => {
```

Comment:
Thanks for taking the time to write tests. One note here, usually we try to test behavior of the component rather than implementation details. This allows more flexibility over time as we may change how the applyKey function works but we know we always want the Select to support options that are JSX elements. Could we change the tests so that we are testing the Select component with JSX elements as options rather than testing the applyKey function?

Reply:
Thanks for the feedback! I've updated the tests to focus on the behavior of the Select component with JSX options, rather than testing the applyKey function directly. Let me know if any further changes are required?